### PR TITLE
Update CIS OCP 1.3.0 section 3

### PR DESCRIPTION
**Pull request by rhmdnd**
This commit fills in the details for CIS OCP 1.3.0 section 3. We have
some rules that check these controls already, so let's reuse them. This
commit also adds some FIXMEs to the controls where we need to update the
CIS benchmark directly.

### CODE REVIEW

File: controls/cis_ocp_1_3_0/section-3.yml [hunk 1]
```diff
@@ -6,27 +6,42 @@ controls:
...
     status: pending
     rules: []
     controls:
+    # FIXME(rhmdnd): The audit steps for this control need to be updated in CIS
+    # upstream so they return relevant configs.
     - id: 3.2.1
       title: Ensure that a minimal audit policy is created
-      status: pending
-      rules: []
+      status: automated
```
Comment (jhrozek) [inline controls/cis_ocp_1_3_0/section-3.yml:31]: same comment as above

File: controls/cis_ocp_1_3_0/section-3.yml [hunk 2]
```diff
@@ -6,27 +6,42 @@ controls:
...
+      rules:
+        - audit_profile_set
       level: level_1
+    # FIXME(rhmdnd): The audit steps for this control need to be updated in CIS
```
Comment (jhrozek) [inline controls/cis_ocp_1_3_0/section-3.yml:35]: same comment as above, except pending should be changed

**Review by jhrozek — CHANGES_REQUESTED** [review]
I think we should rather file cards in https://issues.redhat.com/browse/CMP-1559 for any issues we find in CIS and make use of tickets, notes and the statuses as appropriate.

File: controls/cis_ocp_1_3_0/section-3.yml [hunk 3]
```diff
@@ -6,27 +6,42 @@ controls:
   controls:
   - id: '3.1'
     title: Authentication and Authorization
-    status: pending
+    status: automated
     rules: []
     controls:
+    # FIXME(rhmdnd): This control has some typos in the audit steps that cause
+    # them to not work properly (specifically, escaping characters). This needs
+    # to be fixed in CIS upstream.
```
Comment (rhmdnd) [inline controls/cis_ocp_1_3_0/section-3.yml:14]: Fixing this in https://workbench.cisecurity.org/sections/1652513/recommendations/3216956

File: controls/cis_ocp_1_3_0/section-3.yml (hunk 1 above)
Comment (rhmdnd) [reply controls/cis_ocp_1_3_0/section-3.yml:31]: Addressing this in https://workbench.cisecurity.org/sections/1652514/recommendations/3216957

**Comment by rhmdnd** [comment]
This is ultimately going to be superseded by https://github.com/ComplianceAsCode/content/pull/10749

**Pull request closed without merging**